subsys/fs/Kconfig.fatfs

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2016 Intel Corporation
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config FAT_FILESYSTEM_ELM
+	bool "ELM FAT file system support"
+	depends on FILE_SYSTEM
+	select DISK_ACCESS
+	help
+	  Use the ELM FAT File system implementation.
```

Comment:
@galak Is the `depends on FILE_SYSTEM` what has been missing?

Reply:
Sorry for delay, responded below with what I think is missing.